deploy/koku-migrations-cji.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: koku-migrations-cji
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: koku-migrations-cji
+  spec:
+    database:
+      sharedDbAppName: koku
+    dependencies:
+      - koku
+    envName: ${ENV_NAME}
+    jobs:
+    - name: script-runner-${IMAGE_TAG}
+      podSpec:
+        args:
+        - /bin/bash
+        - -c
+        - $HOME/scripts/run_django_migrations.sh
```

Comment:
Our koku initContainer runs a different script:
https://github.com/project-koku/koku/blob/main/deploy/clowdapp.yaml#L115-L121

I think we should use one or the other in both places.

(We _do_ still need the initContainer in the koku clowdapp so that ephemeral env will run migrations.)